testsuite/grpc-tests/pom.xml

```diff
@@ -123,14 +126,17 @@
         <dependency>
             <groupId>dev.resteasy.grpc</groupId>
             <artifactId>grpc-bridge</artifactId>
+            <version>1.0.0.Alpha6-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>dev.resteasy.grpc</groupId>
             <artifactId>grpc-bridge-runtime</artifactId>
+            <version>1.0.0.Alpha6-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
             <artifactId>resteasy-core</artifactId>
+            <version>6.2.6.Final</version>
```

Comment:
This is a pretty old version at this point. Again, we shouldn't block the PR on this, but we should probably switch to using the `org.jboss.resteasy:resteasy-bom` so we don't need to manage component versions.

Reply:
Still todo. I get errors when I upgrade.